3_javascript/javascript1/week2/js.js

```diff
@@ -0,0 +1,91 @@
+// Flight booking fullname function & Formal fullname
+
+
+function getFullname(firstname , surname, sex=false, useformalname=false){
+    if (useformalname && sex){
+        return "Lady " + firstname + " " + surname;
+    } else if (useformalname) {
+        return "Lord " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+
+const fullname1= getFullname("Youssef", "Topaji" ,false, true);
+const fullname2= getFullname("Benjamin","Hughes", false, false);
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+
+// Weather Wear 
+
+function temperatureOut(temperature){
+    if (temperature > 40){
+        return "Do not go outside if you don't want to be grilled"
+    }
+    else if (temperature > 20){
+        return "You should wear t-shirt and remember to apply sunscreen";
+    } else if (temperature < 20 && temperature > 15){
```

Comment:
I would recommend to change this condition :
`} else if (temperature >= 15 && temperature <= 20) {
        return "You should wear a sweater";`

Reply:
What could be the alternative in this case?
And, is it a bad line?

Reply:
or you mean it would be like 
else if (temperature > 15){
    return "You should wear a sweater";
}

Reply:
Generally, your code here is right, it is me just being picky 🤓
But I would say that solution I proposed ensures that the condition is true when the temperature is 15, 16, 17, 18, 19, or 20,  correctly recommend what to wear in this range temperature. You will ensure here, that when it will be equal to 15 or 20, functionn will work right, in my point of view: :)